federation/pkg/crud/util/crudhelper.go

```diff
@@ -0,0 +1,215 @@
+/*
```

Comment:
ditto.  This was moved from test/integration/federation/framework.  Are you not seeing the code being moved from there to here?